DFC.ServiceTaxonomy.GraphSync/GraphSyncers/ValidateAndRepairGraph.cs

```diff
@@ -69,10 +69,15 @@ public async Task<ValidateAndRepairResult> ValidateGraph()
 
             foreach (ContentTypeDefinition contentTypeDefinition in syncableContentTypeDefinitions)
             {
-                List<ValidationFailure> syncFailures = await ValidateContentItemsOfContentType(contentTypeDefinition, auditSyncLog.LastSynced, result);
-                if (syncFailures.Any())
+                _graphSyncHelper.ContentType = contentTypeDefinition.Name;
+
+                if (!_graphSyncHelper.GraphSyncPartSettings.PreexistingNode)
```

Comment:
Although we don't want to validate the properties of preexisting nodes, we do want to validate the node's relationships.
The properties of the occupation and skill nodes are read-only, but the user can change the associated labels, which changes the relationships off the nodes.
So when validating, we need to validate (and repair if necessary) the outgoing relationships from read-only nodes.

Reply:
Ok, as discussed I've just removed the preexisting check, as the repair just hooks back into a mergeNodeCommand which already checks if the node is preexisting and only updates the relationships if it is.